src/mmw/mmw/settings/production.py

```diff
@@ -39,6 +39,11 @@
 ALLOWED_HOSTS.append(instance_metadata['local-ipv4'])
 # END HOST CONFIGURATION
 
+# FILE STORAGE CONFIGURATION
+DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+AWS_STORAGE_BUCKET_NAME = 'mmw-{}-data-us-east-1'.format('production' if get_env_setting('MMW_STACK_TYPE') == 'Production' else 'staging')
```

Comment:
Looks like a bucket named `mmw-production-data-us-east-1` was created in the staging AWS account. Since bucket names are globally unique, I deleted it so that we can create it in the production account (been waiting for a few minutes, but the name hasn't been made available yet).

When you get around to it tomorrow, please try to recreate the bucket in the production MMW account before merging.